StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1118ParameterMustNotSpanMultipleLines.cs

```diff
@@ -54,7 +57,7 @@ public class SA1118ParameterMustNotSpanMultipleLines : DiagnosticAnalyzer
         /// </summary>
         public const string DiagnosticId = "SA1118";
         private const string Title = "Parameter must not span multiple lines";
-        private const string MessageFormat = "TODO: Message format";
+        private const string MessageFormat = "The parameter spans multiple lines. If the parameter is short, place the entire parameter on a single line. Otherwise, save the contents of the parameter in a temporary variable and pass the temporary variable as a parameter.";
```

Comment:
@macpak That's what the `Description` value is for. The message should be short; many times we reuse the title but occasionally we have a more specific message (e.g. 8b503ee).